_observing-your-data/query-insights/api.md

```diff
@@ -0,0 +1,114 @@
+---
+layout: default
+title: Query Insights API
+parent: Query insights
+nav_order: 50
+---
+
+# Query Insights API
```

Comment:
I would call this as just Query Insights Health API, since the previous "top queries API" should also be considered as a "query insights API". If we want to write this as a new page, we should change the title.

Reply:
I changed to "Query Insights health" and moved both subtopics under this page. Let me know what you think. Thanks!